spec/spec_helper.rb

```diff
@@ -58,4 +58,7 @@
 
   config.before(:all) { ActiveRecord::Base.skip_callbacks = true }
   config.before(:each) { ActionMailer::Base.deliveries.clear }
+
+  config.filter_run focus: true
```

Comment:
Ah cool. Nah, leave it in. I use autotest and a bunch of bash crap to only rerun what's failed / use quickfix windows in vim, and commit history is a mess already. We'll fix everything when we start using Darcs for VCS!